lib/logger.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import logging.config
+
+class SeleniumFilter(logging.Filter):
+    def filter(self, record):
+        selenium_log = 'selenium.webdriver.remote.remote_connection'
+        return selenium_log not in record.getMessage()
+
+# much of this taken from http://victorlin.me/posts/2012/08/26/good-logging-practice-in-python
+# reference -- https://docs.python.org/2/library/logging.config.html
+logging.config.dictConfig({
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "simple": {
+            "format": "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+        },
+        "console": {
+            "format": "%(message)s"
+        }
+    },
+
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+            "level": "INFO",
+            "formatter": "console",
+            "stream": "ext://sys.stdout"
+        },
```

Comment:
There are the handlers. This tells logging info where to go.

`"console"` will not print debug messages, and will print out information to stdout.